HHS/Head-Start-TTADP
# [TTAHUB-1925] Add suspended status to objectives on RTR

## Description of change

Changes to objective suspension and status in general. The Jira contains the pertinent discussion.

1) Objectives can't be suspended from an AR.
2) Complete objectives can't be added to an AR.
3) Objectives that are "Not Started" or "Suspended" when added to an AR will automatically be changed to "in progress" (in the form the user sees, nothing happens in the database until the report is approved).
4) The only valid statuses for an objective on an AR are "In Progress" or "Complete"
5) "Complete" objectives can only have their status change on the RTR (i.e. returned to "In Progress")
6) Add "Suspended" as a status on the RTR. This is the only place objectives will be able to be suspended.

## How to test

Check that you can change an objective status from the RTR, including suspending it, and that you can change it back (until the goal is closed).

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1925

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
